po/zh_CN.po

```diff
@@ -4001,74 +4001,87 @@ msgid ""
 "Column %d of item %d is length %d inconsistent with column %d which is "
 "length %d. Only length-1 columns are recycled."
 msgstr ""
+"第%d列(子项%d)的长度为%d，与第%d列的长度%d不一致。只有length-1列被循环补齐。"
 
 #: rbindlist.c:58
 #, c-format
 msgid ""
 "Column %d ['%s'] of item %d is length 0. This (and %d other%s like it) has "
 "been filled with NA (NULL for list columns) to make each item uniform."
 msgstr ""
+"第%d列['&s'](子项%d的)的长度为0，它(以及%d其他%s类似的)被填充为NA"
+(等同与列表列的NULL值)使得每一个子项统一。"
 
 #: rbindlist.c:62
 #, c-format
 msgid ""
 "Total rows in the list is %lld which is larger than the maximum number of "
 "rows, currently %d"
 msgstr ""
+"列表的总行数为%lld，它比当前行数的最大值%d大。"
 
 #: rbindlist.c:63
 msgid "use.names=TRUE but no item of input list has any names"
 msgstr ""
+"use.names=TRUE，但是输入列表的子项均没有名字。"
 
 #: rbindlist.c:71
 #, c-format
 msgid ""
 "Failed to allocate upper bound of %lld unique column names [sum(lapply(l,"
 "ncol))]"
 msgstr ""
+"分配%lld独有列名[sum(lapply(l,ncol))]的上边界失败。"
```

Comment:
```
无法为 %lld 个独有列名 [sum(lapply(l,ncol))] 的上界分配内存。
```

@MichaelChirico To my understanding, here rbindlist tired to locate as much memory as possible but failed. But `[sum(lapply(l,ncol))]` does not indicate there are any calculations on the unique column numbers. So I got a little bit confused.